maintainers/maintainer-list.nix

```diff
@@ -3908,6 +3908,12 @@
     githubId = 183879;
     name = "Florian Klink";
   };
+  florentc = {
```

Comment:
Please cherry-pick the commit into the PR where you are adding yourself as a maintainer to a package. This makes sure all maintainers entries are actually used and we do not collect stale ones.

Reply:
Would you like me to split the commit into two commits (one to add myself as a maintainer and one to update the package) or open another pull request to add myself as maintainer?

Reply:
That was the wrong template... You can do that in this PR.

That is the right one:

----

Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message ``maintainers: add NAME`` into this PR.

Reply:
This is done.